.github/workflows/publish.yml

```diff
@@ -41,78 +36,70 @@ jobs:
 
   wheel:
     name: Build Wheel
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-20.04
     needs: release
-    permissions: write-all
-
+    container:
+      image: nvidia/cuda:12.1.1-cudnn8-devel-ubuntu20.04
+      options: --gpus all
     strategy:
       fail-fast: false
       matrix:
-          os: ['ubuntu-20.04']
-          python-version: ['3.8', '3.9', '3.10', '3.11']
-          cuda-version: ['11.7']
+        python-version: ['3.8', '3.9', '3.10', '3.11']
+        cuda-version: ['12.1']
 
     steps:
-      - name: Checkout
+      - name: Checkout Source Code
         uses: actions/checkout@v3
 
-    #   - name: Set up Linux Env
-    #     if: ${{ runner.os == 'Linux' }}
-    #     run: |
-    #       bash -x .github/workflows/scripts/env.sh
-
-      - name: Free disk space
+      - name: Free Disk Space
         run: |
           sudo rm -rf /usr/local/cuda-* /opt/cuda
           sudo rm -rf /usr/local/cuda
           bash -x .github/workflows/scripts/free-disk-space.sh
 
-      - name: Set up Python
+      - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v4
         with:
-            python-version: ${{ matrix.python-version }}
+          python-version: ${{ matrix.python-version }}
 
-      - name: Install CUDA ${{ matrix.cuda-version }}
+      - name: Install Dependencies
         run: |
-          bash -x .github/workflows/scripts/cuda-install.sh ${{ matrix.cuda-version }} ${{ matrix.os }}
+          apt update && apt install -y python3-pip rename
+          python3 -m pip install --upgrade pip
+          python3 -m pip install build
 
-      - name: Build wheel
+      - name: Build Wheel
         shell: bash
         run: |
-          sudo apt update && sudo apt install rename -y && sudo apt clean
-          python -m pip install --upgrade pip
-          pip install -r requirements.txt
-          pip install build
-          BUILD_OPS=1 python -m build --wheel
+          BUILD_OPS=1 python3 -m build --wheel
```

Comment:
Should we align the build command to "BUILD_OPS=1 python3 -m build"?

Reply:
Using `BUILD_OPS=1 python3 -m build` builds both the source distribution and the wheel simultaneously. So, I assume this flag is not necessary?